### ISSUE TITLE
Scrub does not run from cron in Ubuntu 16.04

### ISSUE THREAD

**Issue by fantom-x** [zfsonlinux/zfs#4750]
I have noticed that the scrub would never run even though `/usr/lib/zfs-linux/scrub` would start as per the schedule below:
```
cat /etc/cron.d/zfsutils-linux
# Scrub the second Sunday of every month.
24 0 8-14 * * root [ $(date +\%w) -eq 0 ] && [ -x /usr/lib/zfs-linux/scrub ] && /usr/lib/zfs-linux/scrub
```
I have tracked it down to `/usr/lib/zfs-linux/scrub`: it is invoking `zpool` but by default the path is not set in the cron environment in Ubuntu, so it can never find `zpool` tool.

**Comment by kpande** [comment]
please use the mailing list @ http://list.zfsonlinux.org for support questions

**Comment by fantom-x** [comment]
@kpande, How is it a support question if this file `/etc/cron.d/zfsutils-linux` is installed by the `zfsutils-linux` package and it does not work out of the box?

**Comment by kpande** [comment]
because `zfsutils-linux` is a Ubuntu-specific thing, not distributed by zfsonlinux upstream (this project)

**Comment by starling-ib** [comment]
In case Debian users come across this post just as I did: I had this same issue in Debian 8 using jessie-backports and package zfsutils-linux (0.6.5.8-1~bpo8+1). I have reported this bug via https://bugs.debian.org/cgi-bin/pkgreport.cgi?pkg=zfsutils-linux;dist=unstable which I hope is the right place. I'm waiting for the report to be accepted and listed there. I have no idea if the same issue has been reported to the Ubuntu package maintainers.

**Comment by rlaager** [comment]
This was fixed months ago in Ubuntu 16.04:
zfs-linux (0.6.5.6-0ubuntu11) xenial; urgency=medium

  [ Richard Laager ]
  * Set PATH in cron.d job to fix monthly scrubs. (LP: #1548009)

 -- Michael Terry <mterry@ubuntu.com>  Thu, 28 Jul 2016 10:18:25 -0400
